src/components/ImageViewer/ImageViewer.component.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import './ImageViewer.style.css';
+import { useGlobalContext } from '@providers/GlobalContext';
+
+function ImageViewer(props) {
```

Comment:
Seems to me that the wrapper is always the same
```jsx
 <div
    data-testid={ props['data-testid'] }
    className={ 'image-viewer-component ' + className }
>
```
Maybe you can render it as default, and only change the children